dao-utils/src/lib.rs

```diff
@@ -1,8 +1,11 @@
 extern crate alloc;
 
+// Reexport of casper-contract crate.
+pub use casper_contract;
```

Comment:
What's the benefit?

Reply:
In general I would like no to use `casper_contract` directly and use our `casper_env`. It also makes features and dependencies setting a bit cleaner.